demos/browser/app/meetingV2/video/filters/MediaPipeBodySegmentationProcessor.ts

```diff
@@ -0,0 +1,240 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as bodySegmentation from '@tensorflow-models/body-segmentation';
+
+import {
+  CanvasVideoFrameBuffer,
+  VideoFrameBuffer,
+  VideoFrameProcessor,
+} from 'amazon-chime-sdk-js';
+
+
+import { BackgroundImageEncoding } from '../../util/BackgroundImage';
+import { BodySegmenter } from '@tensorflow-models/body-segmentation/dist/body_segmenter';
+import {
+  Segmentation
+} from '@tensorflow-models/body-segmentation/dist/shared/calculators/interfaces/common_interfaces';
+
+const segmenterConfig: bodySegmentation.MediaPipeSelfieSegmentationMediaPipeModelConfig = {
+  runtime: 'mediapipe', // or 'tfjs'
+  solutionPath: 'https://cdn.jsdelivr.net/npm/@mediapipe/selfie_segmentation',
```

Comment:
Isn't there a landscape model too? I couldn't tell if it was much better but seemed more appropriate for demo use case.

Reply:
I tried in their live demo but did not see it is any better.